test/sds/SDS_Regression.ipynb

```diff
@@ -1046,7 +1045,160 @@
+    "                                     variables=['NEE', 'lat', 'lon'],\n",
+    "                                     temporal=hoss_projected_info['temporal_range'])\n",
+    "    submit_and_download(harmony_client, hoss_projected_request, hoss_projected_filename)\n",
+    "    assert exists(hoss_projected_filename), 'Unsuccessful HOSS spatial subset request.'\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "76fed78f",
+   "metadata": {},
+   "source": [
+    "### compare output data with reference data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "97fa4982-7d9c-4dd9-bd4d-21342b20fe2e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import xarray as xr\n",
+    "with xr.open_dataset(hoss_projected_reference_filename) as reference_data, \\\n",
+    "     xr.open_dataset(hoss_projected_filename) as output_data:\n",
+    "    assert output_data.equals(reference_data), 'reference and output datasets did not match'\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "38707fb9-1f9e-4f35-91e4-ed7f07bbfc83",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Set Cartopy features \n",
+    "import cartopy.crs as ccrs\n",
+    "import cartopy.feature as cfeature\n",
+    "\n",
+    "land = cfeature.NaturalEarthFeature(category='physical', name='land', scale='50m',\n",
+    "                                    facecolor=cfeature.COLORS['land'])\n",
+    "ocean = cfeature.NaturalEarthFeature(category='physical', name='ocean', scale='50m',\n",
+    "                                     facecolor=cfeature.COLORS['water'])\n",
+    "\n",
+    "albers_projection = ccrs.AlbersEqualArea(central_longitude=-96.0, central_latitude=40.0,\n",
+    "                                         false_easting=0.0, false_northing=0.0,\n",
+    "                                         standard_parallels=(50.0, 70.0), globe=None)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bc30f67f-21d8-409a-8c1e-8f3b0914961a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(16, 16))\n",
+    "\n",
+    "ax = plt.subplot(1, 2, 1, projection=albers_projection)\n",
+    "ax.add_feature(ocean)\n",
+    "ax.add_feature(land)\n",
+    "ax.gridlines(color='gray', linestyle='dashed', draw_labels=True)\n",
+    "ax.set_extent([-2330000, -1800000, 3900000, 4500000], albers_projection)\n",
+    "output_data.lon.plot.pcolormesh(ax=ax, transform=albers_projection,\n",
+    "                                x='x', y='y', cmap=plt.cm.turbo,\n",
+    "                                add_colorbar=False, zorder=2)\n",
+    "plt.title('Longitude (degrees east)')\n",
+    "\n",
+    "ax = plt.subplot(1, 2, 2, projection=albers_projection)\n",
+    "ax.add_feature(ocean)\n",
+    "ax.add_feature(land)\n",
+    "ax.gridlines(color='grey', linestyle='dashed', draw_labels=True)\n",
```

Comment:
While I'm maybe leaning towards removing plots in the future - nice call to change the grid line colour. The gray is definitely easier to see against the plotted data (I wish I'd used it in my demo).